src/tests/profiler/native/CMakeLists.txt

```diff
@@ -32,9 +32,10 @@ set(SOURCES
 
 include_directories(../../../coreclr/pal/prebuilt/inc)
 
+add_compile_definitions(SOS_INCLUDE)
```

Comment:
This define can use a better name.

Reply:
I'll come back to this and clean this up in a future PR.